app/controllers/person_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PersonController < ApplicationController
+  def index
+    @person = Person.all
```

Comment:
Generally we try to use the plural instance variable for the list of all so `@people`